specs/examples/README.md

```diff
@@ -0,0 +1,89 @@
+# Nimble Examples
+
+## Problem Statement
+
+Nimble should have high-level well-maintained examples for our supported frameworks.
+
+This HLD sketches out the organization and requirements of new examples and of the current examples.
+
+## Links To Relevant Work Items and Reference Material
+
+- [#726 Create a "Getting Started" walkthrough for Nimble Angular](https://github.com/ni/nimble/issues/726)
+- [#325 Set up stackblitz or codepen template with Nimble](https://github.com/ni/nimble/issues/325)
+
+## Implementation / Design
+
+### Frameworks to support
+- Vanilla: TypeScript + SCSS
+    - No explicit examples supporting vanilla JavaScript + CSS with local npm builds. Strongly encourage TS + SCSS.
+    - Examples will use [vite](https://vitejs.dev/)
+        - Supports TS + SCSS out of the box
+        - Web Chapter does not have an alternate recommendation yet for Vanilla apps
+        - Vite works on Stackblitz and local npm builds
+            - Parcel [does not work on Stackblitz](https://github.com/parcel-bundler/parcel/issues/7748)
+            - Rollup requires extra config for TS and SCSS
+            - Webpack aligns with the underlying technology of Angular apps but adds lots of complexity to self manage (Angular generally manages for you)
+- CDN: Vanilla JS and CSS
+    - Uses the CDN builds of packages via unpkg.com
+    - Known issue: fonts are not included in the [existing example](https://github.com/ni/nimble/tree/0fe8cc54f7bd444830e3974969519593b532b083/packages/nimble-components#prototyping-in-a-static-webpage)
+        - We should move all the pre-built cdn bundle stuff to a [separate package](https://twitter.com/justinfagnani/status/1521953590931189760?s=20&t=zejlKVWCbEBce8zYFO7Uyg), ie:
+            - `@ni/nimble-bundle`
+                - alternative names:
+                    - `@ni/nimble-components-cdn`
+                    - `@ni/nimbl-components-bundle`
+                    - `@ni/nimble-cdn-bundle`
+                    - `@ni/nimble-cdn`
+        - Alternative:
+            - We could add a build of our CSS + font assets to the existing nimble-components bundle build which will grow the package size and pose the risk of none cdn users accidentally leveraging the contents (as described in the separate package thread linked above).
+- Angular
+- Blazor
+
+### File structure
+  - examples/
+    - `<framework>`/
```

Comment:
One concern to investigate is the build performance cost of have potentially many small Angular examples. We should verify that the base cost of compiling multiple small examples does not explode build times.

Reply:
We could also consider not having the examples be built as part of the CI build, particularly if they won't be linked off of Storybook.

Reply:
Not as concerned about the CI build as local build. If anything might exclude it from local build but have it run on CI.

Reply:
Another option is to setup one of the task orchestrators discussed in https://github.com/ni/nimble/issues/376

Reply:
Captured as open issue